docs/detections/api/exceptions/api-import-exception-list.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[exceptions-api-import-exception-list]]
+=== Import exception list
+
+Imports an exception list and associated items.
+
+An exception list groups <<exceptions-api-create-exception-item, exception items>>
+and can be associated with rules. When an exception item's query evaluates to
+`true`, rules do *not* issue alerts even when the rule's other criteria are met.
+
+You can assign detection rules with multiple exception lists. For more information, refer to <<rules-api-create>> and <<rules-api-update>>.
+
+IMPORTANT: All exception items added to the same list are evaluated using
+`OR` logic. That is, if any of the items in a list evaluate to `true`, the
+exception prevents the rule from generating an alert. Likewise, `OR` logic is
+used for evaluating exceptions when more than one exception list is
+assigned to a rule. To use the `AND` operator, you can define multiple clauses
+(`entries`) in a single exception item.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists/_import`
+
+The request must include:
+
+* A link to the `.ndjson` file containing the exception list.
+
+For example, using cURL:
+
+[source,console]
+--------------------------------------------------
+curl -X POST "<KibanaURL>/api/exception_lists/_import"
+-u <username>:<password> -H 'kbn-xsrf: true'
+--form "file=@<link to file>" <1>
+--------------------------------------------------
+<1> The relative link to the `.ndjson` file containing the exception list.
+
+===== URL query parameters
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`overwrite` |Boolean |Determines whether existing exception lists with the same
+`list_id` are overwritten and if any exception items have the same `item_id`, those are also overwritten. |No, defaults to `false`.
+|`as_new_list` |Boolean |Determines whether the list being imported will have a new `list_id` generated, and additional `item_id`'s generated for each exception item. Both the exception list container and
+its items are overwritten. |No, defaults to `false`.
+|==============================================
+
+
+===== Example request
+
+Imports the exception list in the `exception_lists.ndjson` file and overwrites
```

Comment:
```suggestion
Imports the exception lists in the `exception_lists.ndjson` file and overwrites
```
Figured this should be plural since the file is called `lists`